lib/src/render.rs

```diff
@@ -310,3 +368,174 @@ where
         text("Set := {"),
     ])
 }
+
+/// creates a body of a typeclass with the given items
+pub(crate) fn new_typeclass_body<'a, I>(items: I) -> Doc<'a>
+where
+    I: IntoIterator,
+    I::Item: pretty::Pretty<'a, pretty::RcAllocator, ()>,
+{
+    intersperse(items, [nil()])
```

Comment:
I think it could be:
```suggestion
    concat(items)
```